### PR TITLE
fix(npm): Export distribution files

### PR DESCRIPTION
**Pull request by Ne3l**
Fixes previous release where dist folder wasn't included on npm distributed version.

### CODE REVIEW

File: .prettierrc [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
```
Comment (ppeszko) [inline .prettierrc:1]: I would remove this setting as this is a subjective change.

Comment (Ne3l) [reply .prettierrc:1]: I got the config from https://github.com/brains-and-beards/mimic-server/blob/master/.eslintrc.js#L22.

Just added the prettierrc with those rules as saving on my editor format the file and then eslint complained (prettier rule) about it.

Comment (Ne3l) [reply .prettierrc:1]: Also i would switch Eslint rules off https://prettier.io/docs/en/integrating-with-linters.html but this is actually a subjective change.

Comment (ppeszko) [reply .prettierrc:1]: I'm actually the other way around, I prefer that the prettier rules are enforced by ESLint. Don't see a good reason for running two tools if they integrate nicely into one.

**Pull request merged**